factcast-site/documentation-docsy/content/en/Setup/metrics.md

```diff
@@ -55,9 +55,9 @@ at [StoreMetrics.java](https://github.com/factcast/factcast/blob/master/factcast
 
 At the **time of writing (0.4.3)**, the metrics exposed by the namespaces group `factcast.server` are:
 
-| operation               | type    | description                      |
-|-------------------------|---------|----------------------------------|
-| handshake               | `timer` | Duration of the GRPC handshakes. |
+| operation               | type    | description                                                                                                                        |
+|-------------------------|---------|------------------------------------------------------------------------------------------------------------------------------------|
+| handshake               | `timer` | Time to process the GRPC handshakes on the application level. Does not include the duration of marshalling and protocol mechanics. |
```

Comment:
:laughing: 

I did not make myself clear. I just suggested to change it to "Duration of the initial handshake", not to confuse people too much with the distinction, as we do not mention the GRPC overhead on the other service-calls as well.